Package.swift

```diff
@@ -37,13 +37,11 @@ var package = Package(
     .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
     .package(url: "https://github.com/ianpartridge/swift-backtrace.git", .exact("1.1.0")),
     .package(url: "https://github.com/vapor/postgres-kit", .exact("2.2.0")),
-    .package(name: "Html", url: "https://github.com/pointfreeco/swift-html.git", .revision("f016529")),
-    .package(
-      name: "Overture", url: "https://github.com/pointfreeco/swift-overture.git", .exact("0.5.0")),
-    .package(name: "Prelude", url: "https://github.com/pointfreeco/swift-prelude.git", .revision("9240a1f")),
+    .package(url: "https://github.com/pointfreeco/swift-html", from: "0.4.0"),
+    .package(url: "https://github.com/pointfreeco/swift-prelude", .revision("7ff9911")),
     .package(name: "SnapshotTesting", url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.8.2"),
-    .package(name: "Tagged", url: "https://github.com/pointfreeco/swift-tagged.git", .revision("fde36b6")),
-    .package(name: "Web", url: "https://github.com/pointfreeco/swift-web.git", .revision("2e23f76")),
+    .package(url: "https://github.com/pointfreeco/swift-tagged", from: "0.6.0"),
+    .package(url: "https://github.com/pointfreeco/swift-web.git", .branch("deprecate-blue-cryptor")),
```

Comment:
When the PR is merged:
```suggestion
    .package(url: "https://github.com/pointfreeco/swift-web", .revision("TODO")),
```